### ISSUE TITLE
Web component: Card section offset - Cards are not operable with keyboard Tab.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6733]
## Detailed description

On FF
Issue:
1. Keyboard Tab focus is not going to the cards so the user is not able to click them.
2. When we navigate with up/down arrow keys, Screen reader announcing links without any label just before the (Eyebrow) text i.e. "label" and at the end of the card. Note visually these links are not operable with the screen reader.

Expected:
1. Tab focus should go to each and every interactive element and it should be keyboard operable.
2. Screen reader is expected to identify the links with appropriate label and the user should be able to activate them.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/293cdff5-b6ba-48c5-bbad-bf32b3b2c26e)

**Comment by Praveen-sr** [comment]
@ariellalgilmore Re tested above issue on both iPhone VO, JAWS and Keyboard focus, Working as expected.
Thanks!
